facebook_sdk/client.py

```diff
@@ -5,10 +5,14 @@
 import requests
 
 from facebook_sdk.constants import BASE_GRAPH_URL, DEFAULT_REQUEST_TIMEOUT
-from facebook_sdk.response import FacebookResponse, FacebookBatchResponse
+from facebook_sdk.response import FacebookBatchResponse, FacebookResponse
 
 
 class FacebookClient(object):
+
+    def __init__(self, request_timeout=None):
+        self.timeout = request_timeout or DEFAULT_REQUEST_TIMEOUT
```

Comment:
This will also set `self.timeout` to the default value, if the provided timeout is zero. I don't think that's a possible use case, so I'm OK with this

Reply:
👍 agree nobody want to wait indefinitely for a response. Thanks!